scapy/layers/tls/cert.py

```diff
@@ -827,10 +827,6 @@ def import_from_asn1pkt(self, crl):
                 date = cert.revocationDate.val
                 if date[-1] == "Z":
                     date = date[:-1]
-                try:
-                    revocationDate = time.strptime(date, "%y%m%d%H%M%S")
```

Comment:
I think this was supposed to test if the date had a correct format or not

Reply:
I will flag it with `# noqa: F841`.

Reply:
I think removing `revocationDate =` should do the trick